src/config/dynamic-theme-fixes.config

```diff
@@ -4647,6 +4647,20 @@ img[src$="layout/logo.svg"]
 
 ================================
 
+freecommander.com
+
+CSS
+.am-body,
+#subsilver-nav-topic {
+    background-image: none !important;
+}
+.header > .row-item,
+.body-blok-h3 h3 {
+    background: #000 !important;
```

Comment:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

Reply:
I thought this had to be done?

Reply:
That's weird... it wasn't showing that I needed to apply the change you made.  I did it now, but the original source is now updated... will it still merge ok?

Reply:
Yup, seems to be fine. Github is sometimes weird, not the first time someone didn't got a email or the correct UI for changes.